nodejs/week3/warmup/index.js

```diff
@@ -23,19 +27,67 @@ router.get("/add", (request, response) => {
             .reduce((accumulator, currentValue) => accumulator.concat(currentValue), [])
             .map((param) => Number(param))
             .reduce((sum, currentNum) => sum + currentNum);
-        isNaN(sumResult) ? response.status(400).send("Invalid input") : response.send(`${sumResult}`);
+        sendResponse(sumResult, response);
+    } catch (error) {
+        response.status(404).send("values undefined");
+    }
+})
+
+router.get("/subtract", (request, response) => {
+    try {
+        const difference = Object.values(request.query)
+            .reduce((accumulator, currentValue) => accumulator.concat(currentValue), [])
+            .map((param) => Number(param))
+            .reduce((previous, current) => previous - current);
+        sendResponse(difference, response);
     } catch (error) {
         response.status(404).send("values undefined");
     }
 })
 
+router.get("/divide", (request, response) => {
```

Comment:
This route should be called `/division`, just like the similar POST route.

Reply:
Thanks Orhan for your time and feedback